### ISSUE TITLE
Not working with SD15 and SD21

### ISSUE THREAD

**Issue by kono-dada** [garibida/ReNoise-Inversion#6]
The model works with SDXL but not works with SD15 or SD21.

When using SD15, the edited image is nearly the same as the original image, rather than a LEGO kitten as shown in the paper. However, SDXL works.

Is this due to any mistake with my `gradio_app.py`? I just change `model_type` and `image_size` to SD15.

``` Python
device = 'cuda' if torch.cuda.is_available() else 'cpu'
model_type = Model_Type.SD15
scheduler_type = Scheduler_Type.EULER
image_size = model_type_to_size(Model_Type.SD15)
pipe_inversion, pipe_inference = get_pipes(model_type, scheduler_type, device=device)
```

**Comment by garibida** [comment]
Hi

We focused on fast diffusion models in the paper, so I didn't try to edit images in that naive way using SD. It looks like you don't have any mistakes, I would try to increase the CFG during inference.

**Comment by lxueaa** [comment]
I tried the following settings, it works for SD2.1:
Firstly, I deal with the v-prediction problem. I delete the line `"prediction_type": "v_prediction"` in `runwayml/stable-diffusion-v1-5/scheduler/scheduler_config.json`, forcing DDIM to use epsilon. Then the code is:
```python
import torch
from PIL import Image
import matplotlib.pyplot as plt

from src.eunms import Model_Type, Scheduler_Type
from src.utils.enums_utils import get_pipes
from src.config import RunConfig
from main import run as invert

device = "cuda:0" if torch.cuda.is_available() else "cpu"
model_type = Model_Type.SD21
scheduler_type = Scheduler_Type.DDIM

pipe_inversion, pipe_inference = get_pipes(
    model_type=model_type,
    scheduler_type=scheduler_type,
    device=device,
)

cfg = RunConfig(
    model_type=model_type,
    scheduler_type=scheduler_type,
    # you can tweak these:
    num_inference_steps=50,      # denoising steps 
    num_inversion_steps=50,      # inversion steps
    guidance_scale=0.0,         # classifier-free guidance during inversion
    num_renoise_steps=7,        # ReNoise iterations per step
    max_num_renoise_steps_first_step=5,
    inversion_max_step=1.0,     # 1.0 = full noise, 0.5 = half steps, etc.
    average_latent_estimations=True,
    average_first_step_range=(0, 5),
    average_step_range=(8, 10),
    noise_regularization_lambda_ac=10.0,
    noise_regularization_lambda_kl=0.055,
    noise_regularization_num_reg_steps=4,
    noise_regularization_num_ac_rolls=5,
    perform_noise_correction=True,
    seed=7865,
)

img_path = "example_images/kitten.jpg"
input_image = (
    Image.open(img_path)
    .convert("RGB")
    .resize((1024, 1024))
)

prompt = "a cat in the photo"
rec_img, inv_latent, noise, all_latents = invert(
    init_image=input_image,
    prompt=prompt,
    cfg=cfg,
    pipe_inversion=pipe_inversion,
    pipe_inference=pipe_inference,
    do_reconstruction=True,
)

fig, axes = plt.subplots(1, 2, figsize=(10, 3))

axes[0].imshow(input_image)
axes[0].set_title("Origin")
axes[0].axis("off")

axes[1].imshow(rec_img)
axes[1].set_title("Reconstruction")
axes[1].axis("off")
```

<img width="748" height="309" alt="Image" src="https://github.com/user-attachments/assets/7d67c911-1a10-48b9-a8d8-069b60ae6d14" />


for SD1.5:

nothing to change, the code is as following:
```python
import torch
from PIL import Image
import matplotlib.pyplot as plt

from src.eunms import Model_Type, Scheduler_Type
from src.utils.enums_utils import get_pipes
from src.config import RunConfig
from main import run as invert

device = "cuda:0" if torch.cuda.is_available() else "cpu"
model_type = Model_Type.SD15
scheduler_type = Scheduler_Type.DDIM

pipe_inversion, pipe_inference = get_pipes(
    model_type=model_type,
    scheduler_type=scheduler_type,
    device=device,
)

cfg = RunConfig(
    model_type=model_type,
    scheduler_type=scheduler_type,
    # you can tweak these:
    num_inference_steps=50,      # denoising steps 
    num_inversion_steps=50,      # inversion steps
    guidance_scale=0.0,         # classifier-free guidance during inversion
    num_renoise_steps=1,        # ReNoise iterations per step
    max_num_renoise_steps_first_step=5,
    inversion_max_step=1.0,     # 1.0 = full noise, 0.5 = half steps, etc.
    average_latent_estimations=True,
    average_first_step_range=(0, 5),
    average_step_range=(8, 10),
    noise_regularization_lambda_ac=10.0,
    noise_regularization_lambda_kl=0.055,
    noise_regularization_num_reg_steps=4,
    noise_regularization_num_ac_rolls=5,
    perform_noise_correction=True,
    seed=7865,
)

img_path = "example_images/kitten.jpg"
input_image = (
    Image.open(img_path)
    .convert("RGB")
    .resize((1024, 1024))
)

prompt = "a cat in the photo"
rec_img, inv_latent, noise, all_latents = invert(
    init_image=input_image,
    prompt=prompt,
    cfg=cfg,
    pipe_inversion=pipe_inversion,
    pipe_inference=pipe_inference,
    do_reconstruction=True,
)

fig, axes = plt.subplots(1, 2, figsize=(10, 3))

axes[0].imshow(input_image)
axes[0].set_title("Origin")
axes[0].axis("off")

axes[1].imshow(rec_img)
axes[1].set_title("Reconstruction")
axes[1].axis("off")
```

<img width="733" height="331" alt="Image" src="https://github.com/user-attachments/assets/229e77ea-0b56-48ab-8555-0167a534f4a6" />


Intuitively, I think v-prediction is not stable in inversion.
